src/BlockInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Dhii\Block;
+
+use Dhii\Util\String\StringableInterface;
+
+/**
+ * Something that can be a block.
+ *
+ * A block is a class that is responsible for generating output.
+ *
+ * @since [*next-version*]
+ */
+interface BlockInterface extends StringableInterface
```

Comment:
Shall we do the same thing as we discussed before, and give this interface a distinct method, such as `getOutput()`? Not sure if this is what we had agreed on, but if you agree with this - feel free to do that.